### PR TITLE
Add Katello 4.14.2 contributors and release notes

### PR DESCRIPTION
**Pull request by chris1984**
#### What changes are you introducing?

Adding in Katello 4.14.2 release notes and updating the contributors

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

For the Katello 4.14.2 release.

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by chris1984** [comment]
@Lennonka how does this look?

File: guides/doc-Release_Notes/topics/katello-4.14.2.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1914[Redmine]
+
+== Katello
+
+=== API
+
+* pass:[apidoc doesn't allow to unset CV/LCE of an activationkey] - https://projects.theforeman.org/issues/38008[#38008]
+
+=== Content Views
```
Comment (Lennonka) [inline guides/doc-Release_Notes/topics/katello-4.14.2.adoc:11]:
```suggestion
=== Content views
```

**Review by Lennonka — COMMENTED** [review]
Sentence case for headings. Otherwise LGTM

File: guides/doc-Release_Notes/topics/katello-4.14.2.adoc (hunk 1 above)
Comment (chris1984) [reply guides/doc-Release_Notes/topics/katello-4.14.2.adoc]: I swear I thought you were on PTO because of your status in Slack 🤣 Updated

**Review by Lennonka — APPROVED** [review]

**Pull request merged**